static/js/main.js

```diff
@@ -42,13 +42,47 @@ $('#CollapseToggleHide').click(function(){
 });
 
 
-//Implementation of Light Mode. Could be handled in a more modern way but it does it's job.
+//Themes. Could be handled in a more modern way but it does it's job.
+$oldbody = null;
+$oldtext = null
+
 $('#ModeToggle').change(function(){
-    $(document.body).toggleClass('LightModeBody');
-    $('.textbox').toggleClass('LightModeTextbox');
-    $('.searchbar').toggleClass('LightModeTextbox');
-    $('.widget').toggleClass('LightModeTextbox');
-    $('.header_header').toggleClass('LightModeTextbox');
+
+    $theme = $(this).val();
+
+    if ($theme == "classic" || $oldbody){

```

Comment:
this code block SHOULD NOT be run at the first change, as $oldbody and $oldtext would be empty at .removeClass making the script not happy, but MUST be run at every change afterwards to clear out the current class, otherwise they would stack, hence why we test if $oldbody isnt empty, and set $oldbody and $oldtext later so that when they are set, the classes will always be flushed